### PR TITLE
fix: Show Stop for UNSTABLE running builds

### PR DESCRIPTION
**Pull request by jithin1987**

### CODE REVIEW

File: tests/integration/components/build-banner/component-test.js [hunk 1]
```diff
@@ -390,3 +390,38 @@ test('it does not render coverage info if there is no coverage step', function (
     assert.notOk($('li').hasClass('coverage'));
   });
 });
+
+test('it should show stop for running UNSTABLE build', function (assert) {
```
Comment (tkyi) [inline tests/integration/components/build-banner/component-test.js]: `it should show the stop button for a running UNSTABLE build` is more clear

Comment (jithin1987) [reply tests/integration/components/build-banner/component-test.js]: updated

File: app/templates/detail/route.js [hunk 2]
```diff
@@ -16,7 +16,9 @@ export default Route.extend({
       tagPayload.forEach((tagObj) => {
         const taggedVerObj = verPayload.find(verObj => verObj.version === tagObj.version);
 
-        taggedVerObj.tag = taggedVerObj.tag ? `${taggedVerObj.tag} ${tagObj.tag}` : tagObj.tag;
+        if (taggedVerObj) {
```
Comment (tkyi) [inline app/templates/detail/route.js]: probably should add this check to commands too

Comment (jithin1987) [reply app/templates/detail/route.js]: added

**Pull request merged**